bindings/matrix-sdk-ffi/CHANGELOG.md

```diff
@@ -31,4 +31,5 @@ Breaking changes:
 
 Additions:
 
+- Add `Room::pin_user_identity`
```

Comment:
I'm sorry, but this can't be exposed in this manner. Please expose the `UserIdentity` and the `pin()` method the same way we have it in the main crate.

Reply:
Done in the force-pushed version I just did